### DISCUSSION TITLE
Dynamic index name

### DISCUSSION THREAD

**Discussion by dfernandesbsolus** [toluaina/pgsync#337] [Q&A]
Hi Guys, it's possible to create an index name with a value from table?

I will explain, i have a table in my database, in this a have a column tenant_id, i need to create one index for each tenant_id, like client_${tenante_id} and put each tenant id data in it specific index, its possible?

**Comment by toluaina** [comment]
- Something similar was requested [here](https://github.com/toluaina/pgsync/issues/67)
- So I'm guessing from your point of view, your use case is that each time you add a new tenant your are automatically populating its associated index.
- This is currently not supported but can be done but will require some dev effort.
